modules/core/shared/src/main/scala/lucuma/core/math/dimensional/coulomb.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math.dimensional
+
+import coulomb.UnitTypeName
+import coulomb.define.BaseUnit
+import coulomb.define.DerivedUnit
+import coulomb.unitops.UnitString
+import lucuma.core.syntax.string._
+import lucuma.core.util.TypeString
+import spire.math.Rational
+
+object BaseUnitDef {
+
+  /** Convenience method to create a coulomb `BaseUnit` and `TypeString` in one go. */
+  def apply[U](name: String = "", abbv: String = "", serialized: String = "")(implicit
+    ut:              UnitTypeName[U]
+  ): BaseUnit[U] with TypeString[U] = {
+    val base = BaseUnit[U](name, abbv)
+
+    val _serialized = if (serialized != "") serialized else base.name.toScreamingSnakeCase
+
+    new BaseUnit[U](base.name, base.abbv) with TypeString[U] {
+      override val serialized: String = _serialized
+    }
+  }
+}
+
+object DerivedUnitDef {
+
+  /** Convenience method to create a coulomb `DerivedUnit` and `TypeString` in one go. */
+  def apply[U, D](
+    coef:        Rational = Rational(1),
+    name:        String = "",
+    abbv:        String = "",
+    serialized:  String
```

Comment:
I thought non default members needed to be declared first

Reply:
Actually, that should have a default value.